### DISCUSSION TITLE
Welcome to electerm Discussions!

### DISCUSSION THREAD

**Discussion by zxdong262** [electerm/electerm#1961] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hinsech475** [comment]
I just installed Electerm v1.21.88 on my Windows 10 64bit OS PC. It appears to be a good candidate to replace my XSHELL 7 as a serial terminal console to various Forth embedded systems. I am hoping to get an end of line delay setting  when uploading files from the PC to the Forth MCUs. The installation completed normally. I am using the default serial settings except for the baud rate which was changed to 115200. When Electerm starts it generates the following error messages. Error 1: An object could not be cloned.
Error: An object could not be cloned. at EventEmitter.i.invoke (node:electron/js2c/renderer_init:57:500) at Object.runGlobalAsync (C:\Program Files\electerm\resources\app.asar\preload\preload.js:101:24) at Hv (http://127.0.0.1:5571/js/electerm.js?1.21.88:2:1848091) at Wv (http://127.0.0.1:5571/js/electerm.js?1.21.88:2:1848301) at http://127.0.0.1:5571/js/electerm.js?1.21.88:2:1860364 Error 2:  error
Opening COM6: Access denied. Question 1: What could be the cause of these errors? Question 2: Will Electerm effectively provide a configurable end of line delay setting when uploading ASCII files from the PC  to the embedded system? Thanks for your help.
